.github/workflows/lint-and-tests.yml

```diff
@@ -27,7 +27,5 @@ jobs:
           go-version: ${{ matrix.go }}
       - run: go mod download
         shell: bash
-      - run: ./scripts/build.sh evm
```

Comment:
Why remove this test? Don't we want to ensure we can build it?

Reply:
I think `build_test` already covers it. but if we want to be sure we can add it again and declare as a dependency for `build_test` run

Reply:
ah nvm these were steps so depending each other already